### PR TITLE
updated header to have release notes button

### PR DESCRIPTION
**Pull request by Adamws33**
added a link to the header to include a link opening in a new tab to the release notes.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,135.237.33.73:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.33.73:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.33.73:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,135.237.33.73,:,80,'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/Header/Header.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const Header = () => {
   const history = useHistory();
   const dispatchRedux = useDispatch();
   const { isAuthenticated, getAccessTokenSilently } = useAuth0();
+  const releaseNotesURL = 'https://precision-sustainable-ag.atlassian.net/wiki/spaces/DST/pages/277413889/Species+Selector+Patch+Notes';
```
Comment (mikahpinegar) [inline src/pages/Header/Header.js:36]: can you put this in the env?

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.33.73:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.33.73:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**